Lib9c/Model/State/CrystalRandomSkillState.cs

```diff
@@ -73,5 +73,12 @@ public IValue Serialize()
 
             return SkillFactory.Get(skillRow, default, 100);
         }
+
+        public int GetHighestRankSkill(CrystalRandomBuffSheet crystalRandomBuffSheet)
+        {
+            return SkillIds
+                .OrderBy(key => crystalRandomBuffSheet[key].Rank)
+                .FirstOrDefault();
+        }
```

Comment:
Why removed `.ThenBy(id => id)` after `.OrderBy(key => ...)`?
If there are Ids that has same `Rank`, this function possible to has nondeterministic issue.

Reply:
Oh it's my mistake. Thx for your comment, I will fix it!